packages/terra-cli/src/terra-dev-site/TerraCLI.tool.mdx

```diff
@@ -0,0 +1,43 @@
+# Terra CLI
+
+Terra CLI is a command line builder that supports creating [yargs commands](https://github.com/yargs/yargs/blob/master/docs/advanced.md#commands) via defining [command modules](https://github.com/yargs/yargs/blob/master/docs/advanced.md#providing-a-command-module). Currently, terra-cli searches for a `terra-cli` directory in a given dependency and assumes that all subdirectories are separate command modules defined via `index.js` files in those subdirectories.
+
+## Allow List
+
+To limit, who can create these commands, terra-cli currently uses an allow list to only search for the following dependencies:
+
+* @cerner/terra-functional-testing
```

Comment:
in the future, are we adding a list for closed source, such as full stack testing?

Reply:
Yeah we would.